cmd/cloud/deployment.go

```diff
@@ -106,10 +111,219 @@ func newDeploymentRootCmd(out io.Writer) *cobra.Command {
 		newDeploymentConnectionRootCmd(out),
 		newDeploymentAirflowVariableRootCmd(out),
 		newDeploymentPoolRootCmd(out),
+		newDeploymentUserRootCmd(out),
+		newDeploymentTeamRootCmd(out),
 	)
 	return cmd
 }
 
+//nolint:dupl
+func newDeploymentTeamRootCmd(out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "team",
+		Aliases: []string{"te", "teams"},
+		Short:   "Manage teams in your Astro Deployment",
+		Long:    "Manage teams in your Astro Deployment.",
+	}
+	cmd.SetOut(out)
+	cmd.AddCommand(
+		newDeploymentTeamListCmd(out),
+		newDeploymentTeamUpdateCmd(out),
+		newDeploymentTeamRemoveCmd(out),
+		newDeploymentTeamAddCmd(out),
+	)
+	cmd.PersistentFlags().StringVar(&deploymentID, "deployment-id", "", "deployment where you'd like to manage teams")
+	return cmd
+}
+
+//nolint:dupl
+func newDeploymentTeamListCmd(out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "list",
+		Aliases: []string{"ls"},
+		Short:   "List all the teams in an Astro Deployment",
+		Long:    "List all the teams in an Astro Deployment",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return listDeploymentTeam(cmd, out)
+		},
+	}
+	return cmd
+}
+
+func newDeploymentTeamRemoveCmd(out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "remove",
+		Aliases: []string{"rm"},
+		Short:   "Remove a team from an Astro Deployment",
+		Long:    "Remove a team from an Astro Deployment",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return removeDeploymentTeam(cmd, args, out)
+		},
+	}
+	return cmd
+}
+
+func listDeploymentTeam(cmd *cobra.Command, out io.Writer) error {
+	if deploymentID == "" {
+		return errors.New("flag --deployment-id is required")
+	}
+	cmd.SilenceUsage = true
+	return team.ListDeploymentTeams(out, astroCoreClient, deploymentID)
+}
+
+func removeDeploymentTeam(cmd *cobra.Command, args []string, out io.Writer) error {
+	if deploymentID == "" {
+		return errors.New("flag --deployment-id is required")
+	}
+	var id string
+
+	// if an id was provided in the args we use it
+	if len(args) > 0 {
+		// make sure the email is lowercase
+		id = args[0]
+	}
+	cmd.SilenceUsage = true
+	return team.RemoveDeploymentTeam(id, deploymentID, out, astroCoreClient)
+}
+
+func newDeploymentTeamAddCmd(out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "add [id]",
+		Short: "Add a team to an Astro Deployment with a specific role",
+		Long:  "Add a team to an Astro Deployment with a specific role\n$astro deployment team add [id] --role [ DEPLOYMENT_ADMIN or the custom role name ].",
```

Comment:
is there usually the spaces there?
just curious

```suggestion
		Long:  "Add a team to an Astro Deployment with a specific role\n$astro deployment team add [id] --role [DEPLOYMENT_ADMIN or the custom role name].",
```

Reply:
removed. the other locations don't add spaces